src/test/java/apicurito/tests/steps/ConfigurationOCPSteps.java

```diff
@@ -86,106 +87,65 @@ public void checkThatApicuritoImageIs(String podType, String value) {
                 break;
             }
         }
-        assertThat(imageName).as("Apicurito has not container from %s", value).isEqualTo(value);
+        assertThat(imageName).as("Apicurito has not container from %s", value).asString().contains(value);
     }
 
     @Then("check that apicurito operator is deployed and in running state")
     public void checkThatApicuritoOperatorIsDeployedAndInRunningState() {
         log.info("Checking that operator is deployed and in running state");
         OpenShiftUtils.xtf().waiters()
-            .areExactlyNPodsReady(1, "name", "apicurito-operator")
+            .areExactlyNPodsReady(1, "name", "fuse-apicurito")
             .interval(TimeUnit.SECONDS, 2)
             .timeout(TimeUnit.MINUTES, 3)
             .waitFor();
     }
 
     @When("deploy operator from operatorhub")
     public void deployOperatorHub() {
+        log.info("Creating apicurito index.");
+        Opm opm = new Opm();
         QuayUser quayUser = new QuayUser(
             TestConfiguration.getQuayUsername(),
             TestConfiguration.getQuayPassword(),
             TestConfiguration.getQuayNamespace(),
             TestConfiguration.getQuayToken()
         );
 
-        QuayService quayService = new QuayService(
-            quayUser,
-            ConfigurationOCPUtils.getOperatorImage(),
-            null
-        );
-        String quayProject = null;
+        Index index = opm.createIndex("quay.io/marketplace/fuse-apicurito-index:" + TestConfiguration.APICURITO_IMAGE_VERSION);
+        Bundle apicuritoBundle = index.addBundle(TestConfiguration.apicuritoOperatorMetadataUrl());
+        index.push(quayUser);
+        OpenShiftService ocpSvc = getOpenShiftService("fuse-apicurito");
+
         try {
-            quayProject = quayService.createQuayProject();
-        } catch (Exception e) {
-            log.error("Could not create quay project", e);
-            fail("Could not create quay project");
+            // OCP stuff - add index
+            ocpSvc.patchGlobalSecrets(TestConfiguration.getQuayPullSecret());
+            index.addIndexToCluster(ocpSvc, "apicurito-test-catalog");
+            log.info("Creating apcurito subscription");
+            apicuritoBundle.createSubscription(ocpSvc);
+        } catch (IOException | TimeoutException | InterruptedException e) {
+            e.printStackTrace();
         }
+    }
 
+    private static OpenShiftService getOpenShiftService(String quayProject) {
         OpenShiftUser adminUser = new OpenShiftUser(
             TestConfiguration.openshiftUsername(),
-            TestConfiguration.openshiftPassword(),
+            TestConfiguration.openshiftUsername(),
```

Comment:
a little confused why username is here twice instead of username and password?

Reply:
lucky mistake :) you are right should be there password instead of username.